__tests__/project.test.js

```diff
@@ -34,3 +34,38 @@ beforeEach(() => {
         });
     })
   });
+
+  describe("GET request along the path of /api/article/:article_id.. returns an object of an article", () => {
+    test(" get an article object with the status code 200", () => {
+      const article_Id=1;
+      return request(app)
+        .get(`/api/article/${article_Id}`)
+        .expect(200)
+        .then((response) => {
+          expect(response.body).toHaveProperty("author", "butter_bridge");
+          expect(response.body).toHaveProperty("title", "Living in the shadow of a great man");
+          expect(response.body).toHaveProperty("body", "I find this existence challenging");
+          expect(response.body).toHaveProperty("topic", "mitch");
+          expect(response.body).toHaveProperty("created_at", expect.any(String));
+          expect(response.body).toHaveProperty("votes", expect.any(Number));
+        });
+    });
+    test(" try to get an article object that does not exist giving a status 404", () => {
+      const article_Id=99999;
+      return request(app)
+        .get(`/api/article/${article_Id}`)
+        .expect(404)
+        .then((response) => {
+          expect(response.body).toEqual({});
+        });
```

Comment:
You shoulfd send back a helpful message here. Something like "article does not exist" would be good!